package.json

```diff
@@ -71,23 +71,23 @@
     "@kie/mock-github": "^1.0.0",
     "@oguzhnatly/react-native-image-manipulator": "github:Expensify/react-native-image-manipulator#5cdae3d4455b03a04c57f50be3863e2fe6c92c52",
     "@onfido/react-native-sdk": "8.3.0",
-    "@react-native-async-storage/async-storage": "^1.17.10",
+    "@react-native-async-storage/async-storage": "^1.19.5",
     "@react-native-camera-roll/camera-roll": "5.4.0",
     "@react-native-clipboard/clipboard": "^1.12.1",
     "@react-native-community/geolocation": "^3.0.6",
-    "@react-native-community/netinfo": "^9.3.10",
+    "@react-native-community/netinfo": "11.1.0",
```

Comment:
Confirmed this is the culprit. Not happen on 9.3.10

Reply:
Seems fixed in latest version (11.2.1)
https://github.com/react-native-netinfo/react-native-netinfo/pull/706 is the fixing PR

Reply:
Should we submit a quick PR with the bump of version or do you want to do it?

Reply:
can you submit if you can? I will handle review

Reply:
Ok, on it now 🚀

Reply:
here it is: https://github.com/Expensify/App/pull/34491